hardhat.config.ts

```diff
@@ -1,3 +1,6 @@
+import * as dotenv from "dotenv"
```

Comment:
FWIW I'm not sure if the use of a .env file for loading environment variables for this config file ever worked properly. In the past, I usually manually set the environment variables in my terminal (i.e. `export ENV_VAR=...`). 

If using `dotenv` here ensures that a .env file can be used then sgtm.